sample/halles_des_vosges/README.md

```diff
@@ -0,0 +1,23 @@
+# Halles des Vosges, Belfort, France
+
+This sample is a demonstration of multi-material building and slopped roof.
+
+The construction method is based on the building divided in sectors. These sectors are specified by their angles, and have no fixed amount of sides, in the *.mif* and *.mid* files. The slopes are specified in each *.ext* file. In this case 18 files are sufficient to describe all 28 sectors.
+
+![](./sectors.png)
+\
+*Figure 1 : Sectors*
+
+With the *.ext* file format you can reduce to a certain minimum the number of files you need, because this building specific building has many slopped roof, all different slope needs to have its own description.
```

Comment:
What does "building specific building" mean ?

Reply:
This sentence does not seem to bring any new info compared to the one on line 5.